jsonschema-maven-plugin/src/test/java/com/github/victools/jsonschema/plugin/maven/SchemaGeneratorMojoTest.java

```diff
@@ -75,7 +75,11 @@ public void testGeneration(String testCaseName) throws Exception {
         File referenceFile = new File(testCaseLocation, testCaseName + "-reference.json");
         Assertions.assertTrue(referenceFile.exists());
         Assertions.assertTrue(FileUtils.contentEqualsIgnoreEOL(resultFile, referenceFile, CHARSET_NAME),
-                "Generated schema for " + testCaseName + " is not equal to the expected reference.");
+                "Generated schema for " + testCaseName + " is not equal to the expected reference.\n" +
+                        "Generated: " + FileUtils.readFileToString(resultFile) + "\n" +
+                        "Expected: " + FileUtils.readFileToString(referenceFile));
+        System.out.println(FileUtils.readFileToString(resultFile));
+        System.out.println(FileUtils.readFileToString(referenceFile));
```

Comment:
**suggestion**: Can remove the sysouts again, now that the tests are succeeding again. 😃
The extra detail in the printout for a failed test is fine, but perhaps add some extra line breaks, to not start on the line saying "Generated:" or "Expected:" but on the one below.
```suggestion
                "Generated schema for " + testCaseName + " is not equal to the expected reference.\n"
                        + "Generated:\n"
                        + FileUtils.readFileToString(resultFile)
                        + "\n----------\n"
                        + "Expected:\n"
                        + FileUtils.readFileToString(referenceFile));
```